internal/graph/cached_resolver.go

```diff
@@ -139,10 +140,13 @@ func (c *CachedCheckResolver) ResolveCheck(
 	ctx context.Context,
 	req *ResolveCheckRequest,
 ) (*ResolveCheckResponse, error) {
-	ctx, span := tracer.Start(ctx, "ResolveCheck")
+	ctx, span := tracer.Start(ctx, "ResolveCheck", trace.WithAttributes(
```

Comment:
I feel inclined to change the span name to `CachedCheckResolver.ResolveCheck`, `LocalChecker.ResolveCheck`, etc. WDYT?

Reply:
Sounds good to me 👍🏻

Reply:
I'll send a different PR.